app/common/resources/public/language/en.edn

```diff
@@ -439,7 +441,7 @@
   :task.type/eia-pre-assessment "EIA pre-assessment",
   :task.type/owners-supervision "Owners' supervision",
   :task.group/study "Studies",
-  :file.status/draft "Draft",
+  :file.status/draft "permission-already-granted",
```

Comment:
Not sure if this is something that tags along from my commit, but looks strange.

Reply:
looks like a mistake in localization excel

Reply:
@solita-erkkiv remove it from the Excel, it'll be removed from the localizations on the next import.